packages/stream_chat_flutter/lib/src/attachment/attachment_upload_state_builder.dart

```diff
@@ -27,7 +27,9 @@ class AttachmentUploadStateBuilder extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (attachment.uploadState == null) return Offstage();
+    if (message.status == null || message.status == MessageSendingStatus.sent) {
+      return Offstage();
```

Comment:
Do you need an `Offstage` here? Would a `SizedBox` be better for a placeholder widget?

Reply:
I don't think it makes much difference
We use it also in other places